Formula/impute2.rb

```diff
@@ -0,0 +1,25 @@
+class Impute2 < Formula
+  desc "Genotype imputation and haplotype phasing program"
+  homepage "https://mathgen.stats.ox.ac.uk/impute/impute_v2.html"
+
+  # tag "bioinformatics"
+  # doi "10.1371/journal.pgen.1000529"
+
+  version "2.3.2"
+  if OS.mac?
```

Comment:
No `OS.mac?`, we assume we are on macOS

Reply:
This was originally in homebrew-science, so I don't know if that supports Linuxbrew... Is homebrew-core not designed for Linuxbrew?